### ISSUE TITLE
Onion skinning

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-paint#904]
Transparent overlays of neighboring costumes to make animating easier

**Comment by DeleteThisAcount** [comment]
I like this idea, making scratch a much more versatile animation tool.

**Comment by Explosion-Scratch** [comment]
Yes this would be helpful for animations, currently one has to copy/paste the previous frame which can interfere with the editing process and be confusing as to which is the original and which is the previous frame.

**Comment by jeffalo** [comment]
https://user-images.githubusercontent.com/40470736/103513451-ef1bab80-4e6a-11eb-9fd0-4c2c663300ec.mp4

this is how https://github.com/ScratchAddons/ScratchAddons/pull/879 does it, perhaps if this is still planed, you can take inspiration from the design on that. @fsih

**Comment by Crispydogs101** [comment]
I support this because this would be a boost up for new animators
